### PR TITLE
RCHAIN-3483 add pre-defined bond file

### PR DESCRIPTION
**Pull request by zsluedem**
## Overview
add pre-defined bond file

### JIRA ticket:
https://rchain.atlassian.net/browse/RCHAIN-3483

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Comment by zsluedem** [comment]
bors try

**Comment by zsluedem** [comment]
use futures;

**Comment by zsluedem** [comment]
bors try

File: integration-tests/test/conftest.py [hunk 1]
```diff
@@ -120,14 +119,20 @@ def random_generator(command_line_options: CommandLineOptions) -> Generator[Rand
 
 
 @contextlib.contextmanager
-def testing_context(command_line_options: CommandLineOptions, random_generator: Random, docker_client: DockerClient, bootstrap_key: PrivateKey = None, peers_keys: List[PrivateKey] = None, network_peers: int = 2) -> Generator[TestingContext, None, None]:
+def testing_context(command_line_options: CommandLineOptions, random_generator: Random, docker_client: DockerClient, bootstrap_key: PrivateKey = None, peers_keys: List[PrivateKey] = None, network_peers: int = 2, validator_bonds_dict: Dict[PrivateKey, int] = None) -> Generator[TestingContext, None, None]:
```
Comment (adaszko) [inline integration-tests/test/conftest.py:122]: The `validator_bonds_dict` parameter isn't used anywhere, right?

Comment (zsluedem) [reply integration-tests/test/conftest.py:122]: yes~

Comment (adaszko) [reply integration-tests/test/conftest.py:122]: So I'd say this PR is too small to be merged.  It only makes sense to merge it once the added code is in use.  Please push more changes to it related to whatever you're aiming at.

**Pull request closed without merging**